### PR TITLE
CAL-224 added docs for neutral taxonomy

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

adds documentation for neutral taxonomy to `draft-documentation`

#### Who is reviewing it?

@brendan-hofmann @emanns95 @AzGoalie
#### Choose 2 committers to review/merge the PR.

@jlcsmith
@kcwire
@pklinef
@shaundmorris

#### How should this be tested?

- review content of tables
- builds successfully.

#### What are the relevant tickets?

[CAL-224](https://codice.atlassian.net/browse/CAL-224)
[DDF-2682](https://codice.atlassian.net/browse/DDF-2682)

#### Screenshots (if appropriate)

[draft-documentation.pdf](https://github.com/codice/alliance/files/712000/draft-documentation.pdf)

#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+|
+
+|isr.organizational-unit
+|An organizational unit.
+|String
+|< 1024 characters
+|
+
+|isr.niirs
+|The quality or detail level of an image. (National Imagery Interpretability Rating Scale)
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: Should all these definitions end in a period? Seems like we are inconsistent throughout the tables on it.

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+|isr.category
+|A STANAG 4559 image or video category.
+|String
+|
+|
+
+|isr.image-id
+|An ISR imagery identifier.
+|
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: Is there a reason why these don't have a type? We may want to make a note if there is a reason.

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+|isr.platform-name
+|An ISR platform name.
+|
+|
+|
+
+|isr.exploitation-level
+|The degree of exploitation performed on the original data. A value of '0' means that the product is not exploited.
+|Integer
+|0..9
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: Is there a reason cloud clover is `0-10` and this is `0...9`?

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc (hunk 1 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: They should be consistent. This is a port of the rough cut on the wiki.

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc (hunk 2 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: The wiki was incomplete. We will need to refine this after it is merged.

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc (hunk 3 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: @jckilmer may recall better, but I believe we adopted from existing standards where possible. I agree it looks funky, and at the very least the format of the ranges should be consistent.

File: distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc]: https://codice.atlassian.net/browse/DDF-2686 is the task to go beyond what's on Confluence

**Pull request merged**